test/cl_run.clas.abap

```diff
@@ -40,7 +41,8 @@ CLASS cl_run IMPLEMENTATION.
       ENDIF.
     ENDLOOP.
 
-    CONCATENATE LINES OF string_tab INTO result SEPARATED BY |\n|.
+    newline = |\n|.
+    CONCATENATE LINES OF string_tab INTO result SEPARATED BY newline.
```

Comment:
Is this changer needed? Is transpiler facing issues with the old syntax?

Reply:
yea, it was not valid ABAP syntax

Reply:
I see, you are right.